### DISCUSSION TITLE
segment-anything-2 implementation is already there

### DISCUSSION THREAD

**Discussion by hoodady** [Panchovix/stable-diffusion-webui-reForge#95] [Ideas]
https://github.com/facebookresearch/segment-anything-2

It shows the below new models in the list but they don't work for me.

sam2_hiera_tiny.pt
sam2_hiera_small.pt
sam2_hiera_base_plus.pt
sam2_hiera_large.pt

**Comment by wm19371015100-cyber** [comment]
+1
